packages/api-client/src/components/Sidebar/SidebarListElement.vue

```diff
@@ -89,9 +89,9 @@ const handleRename = (id: string) => {
         class="text-sidebar-c-2 size-3.5 stroke-[2.25]"
         :icon="variable.icon" />
       <span
-        class="empty-variable-name text-sm line-clamp-1 break-all group-hover:pr-5"
-        >{{ variable.name }}</span
-      >
+        class="empty-variable-name text-sm line-clamp-1 break-all group-hover:pr-5">
```

Comment:
where do these come from, is this prettier

Reply:
I was just fixing it

Reply:
yea I know 😄

Reply:
Oh yeah our prettier is 100% deterministic I can't remember why, I believe this is our preferred indentation though